store/tikv/mock-tikv/cop_handler.go

```diff
@@ -103,25 +99,27 @@ func (h *rpcHandler) handleCopRequest(req *coprocessor.Request) (*coprocessor.Re
 				}
 			}
 		}
-		ctx.aggregate = len(sel.Aggregates) > 0 || len(sel.GetGroupBy()) > 0
-		if ctx.aggregate {
+		aggregate := len(sel.Aggregates) > 0 || len(sel.GetGroupBy()) > 0
+		if aggregate {
```

Comment:
I think we don't need to change the old code. It will make the refactor more complex.